### PR TITLE
Feature: cloaked shield permeability

### PR DESCRIPTION
**Pull request by Quantumshark**
**New feature**
https://github.com/endless-sky/endless-sky/pull/10610

### CODE REVIEW

File: wiki/CreatingOutfits.md [hunk 1]
```diff
@@ -131,6 +131,8 @@ Unless otherwise stated, other outfit attributes will stack additively between m
 
   * `"low shield permeability"`: The permeability of your shields as they approach 0%. A shield which is permeable allows some damage to bleed through to the hull. For example, a permeability of 10% means that 90% of the damage hits the shields and 10% hits the hull. As shield strength increases, the permeability of your shields approaches the high shield permeability value. **(v. 0.10.1)**
 
+  * `"cloaked shield permeability"`: The permeability of your shields is increased by this much while you are cloaked. A shield which is permeable allows some damage to bleed through to the hull. For example, a permeability of 10% means that 90% of the damage hits the shields and 10% hits the hull. This stacks additively with the two permeability attributes above. **(TBD: add version number here when main PR is to be merged)**
```
Comment (TheGiraffe3) [inline wiki/CreatingOutfits.md:134]: Per second, or how often?

Comment (Quantumshark) [reply wiki/CreatingOutfits.md:134]: I don't understand what you mean.

Comment (Quantumshark) [reply wiki/CreatingOutfits.md:134]: No.
While you are cloaked, it is increased by this much.
So, for example, with a cloaked shield permeability of 0.1 and 20% permeability from low and high shield permeability, your permeability is 30% while you are cloaked, and 20% while you are uncloaked.

**Review by TheGiraffe3 — APPROVED** [review]
I'm guessing it'll probably be merged in 0.10.11.

File: wiki/CreatingOutfits.md (hunk 1 above)
Comment (Quantumshark) [reply wiki/CreatingOutfits.md:134]:
```suggestion
  * `"cloaked shield permeability"`: The permeability of your shields is increased by this much while you are cloaked. A shield which is permeable allows some damage to bleed through to the hull. For example, a permeability of 10% means that 90% of the damage hits the shields and 10% hits the hull. This stacks additively with the two permeability attributes above. **(v. 0.10.11)**
```

**Review by tibetiroka — APPROVED** [review]

**Pull request merged**